COVID19/src/screens/Prevention.js

```diff
@@ -12,33 +13,58 @@ const Prevention = ({navigation}) => {
       pages={[
         {
           backgroundColor: colors.purpleMedium,
-          image: <Image source={require('../utils/images/mask.png')} />,
+          image: (
+            <Image
+              style={styles.imageSize}
+              source={require('../utils/images/mask.png')}
+            />
+          ),
           title: constans.wearMask,
+          subtitle: constans.wearMaskSub,
         },
         {
           backgroundColor: colors.purpleLight,
-          image: <Image source={require('../utils/images/hands.png')} />,
+          image: (
+            <Image
+              style={styles.imageSize}
+              source={require('../utils/images/hands.png')}
+            />
+          ),
           title: constans.washHands,
+          subtitle: constans.washHandsSub,
         },
         {
           backgroundColor: colors.purpleDark,
-          image: <Image source={require('../utils/images/distance.png')} />,
+          image: (
+            <Image
+              style={styles.imageSize}
+              source={require('../utils/images/distance.png')}
+            />
+          ),
           title: constans.keepDistance,
+          subtitle: constans.keepDistanceSub,
         },
         {
           backgroundColor: colors.purpleMedium,
-          image: <Image source={require('../utils/images/sanitizer.png')} />,
+          image: (
+            <Image
+              style={styles.imageSize}
+              source={require('../utils/images/sanitizer.png')}
```

Comment:
Try to import all the images at the file's beginning

Reply:
fixed